packages/tf-inventory/src/stateFile.js

```diff
@@ -95,20 +95,15 @@ async function listTfStateFiles() {
 async function listClusterEC2Instances(clusterArn) {
   const clusterContainerInstances = await aws.ecs().listContainerInstances({
```

Comment:
Does this need to be updated?

Reply:
What's the reasoning for not updating this? Right now stateFile.js pulls in all of `@cumulus/aws-client/services` but it appears to only be using dynamo, s3, and ecs.

Reply:
I see there's a reverted change that would have imported only what we need:
https://github.com/nasa/cumulus/commit/6b6bbf9e4b1706c0b187b152b7ca55b1272dd637

What's the reasoning for reverting? Is it because we stub `aws`?